docs/bosh/README.md

```diff
@@ -398,13 +451,16 @@ Now you have the infrastructure ready to deploy a BOSH director.
 terminal multiplexer such as `tmux` or `screen`.
 
    ```
-   bosh-init deploy manifest.yml
+   bosh2 create-env manifest.yml
    ```
 
-1. Target your BOSH environment:
+1. Download [ca_cert.pem](../ca_cert.pem) from this repository.
```

Comment:
this doc should be copy & paste-able, can you use `bosh2 int` to extract & write the ca_cert value?